package.json

```diff
@@ -161,6 +161,10 @@
         "command": "coverage-gutters.displayCoverage",
         "title": "Coverage Gutters: Display Coverage"
       },
+      {
+        "command": "coverage-gutters.toggleCoverage",
+        "title": "Coverage Gutters: Toggle Coverage"
+      },
```

Comment:
You might also want to add a key binding, but I am not sure which you might use 🤔.
https://github.com/ryanluker/vscode-coverage-gutters/pull/385/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L177-L204

Reply:
Yes I thought about it. How about `ctrl/cmd+shift+5`?

Reply:
Added this stanza for now, feel free to tweak:

```json
{
  "command": "coverage-gutters.toggleCoverage",
  "key": "ctrl+shift+5",
  "mac": "shift+cmd+5"
}
```

Reply:
Looks good to me! I think the only keybinding we can't use is ctrl+shift+0 due to vscode not seeing it for some reason (based on a previous ticket).